ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-release-0.34.yaml

```diff
@@ -55,7 +47,11 @@ tests:
 
         DOCKER_PREFIX='kubevirt'
         export DOCKER_PREFIX
-        DOCKER_TAG="v0.33.0"
+        DOCKER_TAG=$(
```

Comment:
I haven't tested this bash code to check it's output. Is there a reasonable default we could use in case this fails to parse anything meaningful?

Reply:
```
> curl --fail -s https://api.github.com/repos/kubevirt/kubevirt/releases |
                  jq -r '(.[].tag_name | select( test("-(rc|alpha|beta)") | not ) )' |
                  sort -rV | grep 'v0.34' | head -1
```
yields
```
v0.34.0
```

ATM. We could just go with hard coding whatever the latest z release is but I figured this would be tedious.